segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/DefragmentOperation.java

```diff
@@ -145,21 +145,19 @@
                             } else {
                                 f = CompletableFuture.completedFuture(null);
                             }
-                            return f.thenApplyAsync(vv -> {
+                            return f.thenRunAsync(() -> {
```

Comment:
This is another example of when `Async` is expected to cause more perf harm than not.

Reply:
same as above.